File: App/API/Entities/Products/Whiskey.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using API.Entities.Products;

namespace API.Entities
{
    public class Whiskey
    {
        public int Id { get; set; }
        
        public string Name { get; set; }
        
        public int Age { get; set; }
        
        public string Bottler { get; set; }
        
        public string Type { get; set; }
        
        public string PhotoUrl { get; set; }
        
        public double Price { get; set; }
        
        public string Region { get; set; }
        
        public float Rating { get; set; }
        
        public int Rating_Count { get; set; }
        
        public string Cask_Type { get; set; }

        [NotMapped]
        public string[] Characteristics { get; set; }
        
        public string Colouring { get; set; }
        
        public string Age_Statement { get; set; }
        
        public double Proof { get; set; }

        public Style Style { get; set; }
        
        public int Quantity { get; set; }

        public double CurrentUserRating { get; set; }
        
        public DateTime DateRated { get; set; }
    }
}

File: App/client/src/app/models/whiskey.ts
import { Style } from "./style"
import { Rating } from './rating';

export interface Whiskey {
  id: number,
  name: string,
  age: number,
  bottler: string,
  type: string,
  photoUrl: string,
  price: number,
  region: string
  rating: number,
  rating_Count: number,
  cask_Type: string,
  characteristics: string[],
  colouring: string,
  age_Statement: string,
  proof: number,
  style: Style,
  quantity: number
  currentUserRating: number
  dateRated: Date
}
